### ISSUE TITLE
As an admin, I want changes made to people and events in the admin interface to automatically update the member search, so that content in the search and admin interface stay in sync.

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/mep-django#339]
# testing notes
When you edit records in the django admin, they should be automatically updated in the Solr index so that data edits are reflected in the public site.  For each of the following, please test by making changes in the django admin and then reloading the corresponding public search/browse page to see that your changes have been propagated.
- members
  - edit fields on the person record (e.g. sort name)
  - edit event dates associated with an account + person
  - add/remove card on an account
  - delete an event
  - delete an account
  - delete a person

## dev notes
- [x] nationality depends on country
- [x] has_card depends on account
- [x] account_years depends on account__events

Split out into #425 (books) and #426 (cards).

**Comment by rlskoeser** [comment]
Porting from PPA required adapting and abstracting, and it doesn't cover all our cases for related model indexing. Increasing from 3 to 5.

**Comment by clmahoney** [comment]
- YES, I am able to edit fields on the person record - I gave "Abbott" a first name as a sort name: Abbott, Lily, and that appeared on the site! Then I changed it back
- YE, I can edit event dates associated with an account and person. I changed "Accursi"'s membership dates from within 1941 to 1930-1941, and that appeared on the site and one the member timeline
- YES I can delete an account AND a person -- I deleted Achenbach, and then I added Achenbach BACK
- YES, I can delete an event - I deleted the subscription of "Abdul" and that was mirrored on the site! 

I wasn't sure how to "add or remove a card on an account" ?
